shared/menubar/user-add.desktop.js

```diff
@@ -25,22 +25,23 @@ const UserButton = ({isPublic, onClick}: {isPublic: boolean, onClick: () => void
   </Box>
 )
 
-const UserInput = ({isPublic, onSubmit, onCancel, onUpdateText, username}) => {
+const UserInput = ({isPublic, onSubmit, onCancel, onUpdateText, username, text}) => {
   const icon: IconType = isPublic ? 'icon-folder-public-open-32' : 'icon-folder-private-open-32'
 
   return (
     <Box style={{...stylesInputContainer,
       backgroundColor: isPublic ? globalColors.lightGrey : globalColors.darkBlue3}}>
-      {!isPublic && <Text type='BodySemiboldItalic' style={stylesPrivatePrefix}>{username},</Text>}
       <Input
         small={true}
```

Comment:
A boolean for small seems weird. Maybe make this a type?
And then use label, labelStyle instead of smallLabel, smallLabelStyle.